### PR TITLE
More general readers and writers

### PR DESCRIPTION
**Pull request by martinpopel**
Use e.q. `read.Conllu files='!in[12]*.conllu'`

### CODE REVIEW

File: udapi/block/read/conllu.py [hunk 1]
```diff
@@ -1,22 +1,17 @@
-import logging
 import re
-import bz2
 
 from udapi.core.basereader import BaseReader
 from udapi.core.root import Root
 
+# Compile a set of regular expressions that will be searched over the lines.
+RE_SENT_ID = re.compile(r'^# sent_id\s*=\s+(\S+)')
```
Comment (VincTheSecond) [inline udapi/block/read/conllu.py]: Why did you add the `=` char to this pattern? All treebanks used by my project (CzEng, UDv1.4, ...) use
`# sent_id <sent_id>`

Comment (martinpopel) [inline udapi/block/read/conllu.py]: The equal sign was added in UD v2.0 [specification](http://universaldependencies.org/format.html).
Now, I've made the equal sign optional in the reader to support pre-v2 files.
